### ISSUE TITLE
Implement `size` field for capped collections

### ISSUE THREAD

**Issue by b1ron** [FerretDB/FerretDB#2813]
### What should be done?

Depends on #75.

* https://www.mongodb.com/docs/manual/reference/command/create/#command-fields

> integer | Optional. Specify a maximum size in bytes for a capped collection. Once a capped collection reaches its maximum size, MongoDB removes the older documents to make space for the new documents. The size field is required for capped collections and ignored for other collections.

**Comment by AlekSi** [comment]
Let's use #75 for that

**Comment by AlekSi** [comment]
Duplicate of #75
